app/game/game.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+class WebGame
+  GAME_INSTANCE_VARIABLES = %i[@code @user @stage @difficulty @possible_hints].freeze
+
+  attr_reader :codebraker_game
+
+  def initialize(request)
+    @request = request
+    @codebraker_game = Codebraker::Game.new
```

Comment:
```suggestion
    @codebraker_game = Codebraker::Game.new(user: user, difficulty: difficluty)
    
    or 
    
    add configure_game method to gem
```

Reply:
generate random string and save it to session,
add module/class to save instance to file